src/pages/hoverboard.tsx

```diff
@@ -0,0 +1,79 @@
+import Head from 'next/head';
+import { useRef } from 'react';
+import styled from 'styled-components';
+
+const colors = ['#e74c3c', '#8e44ad', '#3498db', '#e67e22', '#2ecc71'];
+
+function getRandomColor() {
+  return colors[Math.floor(Math.random() * colors.length)];
+}
+
+interface BoxProps {
+  ind: number;
+}
+
+function Box({ ind }: BoxProps) {
+  const cardRef = useRef<any>();
+
+  function setColor() {
+    const color = getRandomColor();
+    cardRef.current.style.background = color;
+    cardRef.current.style.boxShadow = `0 0 2px ${color}, 0 0 10px ${color}`;
+  }
+
+  function removeColor() {
+    cardRef.current.style.background = '#1d1d1d';
+    cardRef.current.style.boxShadow = '0 0 2px #000';
+  }
+
+  return <Card key={ind} ref={cardRef} onMouseOver={setColor} onMouseOut={removeColor} />;
```

Comment:
Instead of using onMouseOver, use onMouseEnter

Reply:
also this @sank2000